content/authentication/guides/systemvendor/_index.en.md

```
@@ -0,0 +1,206 @@
+  - Refer [here](../../../api/authentication/systemregister/model/) for the detailed description for information about each input. For all subsequent communications, the system vendor must reference the value specified in the 'id' field.
+
+### Creating a System User For Standard System User
+The system user can be created in two ways
+
+  1. End user driven
+      - Smartcloud As must request "TILFELDIG SUBTIL APE" (end user) to login to altinn portal and create a system user for their system SmartCloud
+  2. Vendor driven
+      - SmartCloud AS creates a system user request (via altinn api for system user request) from within SmartCloud, which is sent to Altinn.
+      For detailed information on each input field, refer the documentation [here](../../../api/authentication/systemuserrequest/external/model/)
+
+      ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)